lib/chef/knife/azure_server_create.rb

```diff
@@ -69,8 +69,7 @@ def load_winrm_deps
 
       option :ssh_port,
         :long => "--ssh-port PORT",
-        :description => "The ssh port. Default is 22.",
-        :default => '22'
+        :description => "The ssh port. Default is 22. If --azure_connect_to_existing_dns set then default SSH port is random"
```

Comment:
In this case, we're mixing the '--' with the underscores -- on the cli, underscores aren't used (they are used in config files). So we should be consistent -- use the name as it would appear in a config file, or just as it would appear on the cli, but not a hybrid.